app/src/routes/api/artists/+server.ts

```diff
@@ -17,5 +17,12 @@ export async function GET() {
 		return json({ error: 'Failed to fetch artist data' }, { status: 500 });
 	}
 
+	// Sort artists by name
+	data.sort((a, b) => {
```

Comment:
Possibly not the place for this but feels orderly to have artists in alphabetical order by default.